executor/executor_test.go

```diff
@@ -2088,6 +2088,21 @@ func (s *testSuite) TestTiDBCurrentTS(c *C) {
 	c.Assert(terror.ErrorEqual(err, variable.ErrReadOnly), IsTrue)
 }
 
+func (s *testSuite) TestInsertDecimalTruncated(c *C) {
```

Comment:
you can put this test in `executor/write_test.go:34:func (s *testSuite) TestInsert(c *C) `